sound/soc/sof/debug.c

```diff
@@ -338,6 +339,13 @@ int snd_sof_dbg_init(struct snd_sof_dev *sdev)
 	/* use "sof" as top level debugFS dir */
 	sdev->debugfs_root = debugfs_create_dir("sof", NULL);
 
+	debugfs_create_str("fw_path", 0444, sdev->debugfs_root, (char **)&plat_data->fw_filename_prefix);
+	debugfs_create_str("fw_lib_path", 0444, sdev->debugfs_root, (char **)&plat_data->fw_lib_prefix);
+	debugfs_create_str("tplg_path", 0444, sdev->debugfs_root, (char **)&plat_data->tplg_filename_prefix);
+	debugfs_create_str("fw_name", 0444, sdev->debugfs_root, (char **)&plat_data->fw_filename);
+	debugfs_create_str("tplg_name", 0444, sdev->debugfs_root, (char **)&plat_data->tplg_filename);
+	debugfs_create_u32("ipc_type", 0444, sdev->debugfs_root, (u32 *)&plat_data->ipc_type);
```

Comment:
Should we have a folder to hold these? /sys/kernel/debug/sof/ starts to be a bit busy.

Reply:
@ujfalusi we can add a folder, but what would the name be? 'fw_profile' or something?

Reply:
@plbossart, you got me ;)
it is not just paths and filenames but the ipc_type as well and it is not really a FW configuration.
`profile` or `driver profile`, these dictates what firmware and what IPC to use. `platform profile`?

I'm really bad at this.

Reply:
"ci_profile" ?

Reply:
it is not a ci profile, I would say, it is a platform_profile, or platform_config?
On cAVS2.5 we can have IPC3 or IPC4 depending on what is selected, selected_config, selected_profile?

Reply:
Both `fw_config` and `fw_profile` work for me.

`fw_path` is technically not part of the firmware configuration, but the reason why we have different `fw_path` is because they point to different firmware configurations.

Reply:
Any objections on "fw_profile" then?